plugins/inputs/opcua/opcua_test.go

```diff
@@ -437,3 +437,177 @@ use_unregistered_reads = true
 	require.EqualValues(t, map[string]string{"tag1": "override", "tag2": "val2"}, o.client.NodeMetricMapping[3].MetricTags)
 	require.EqualValues(t, map[string]string{"tag1": "val1", "tag2": "val2"}, o.client.NodeMetricMapping[4].MetricTags)
 }
+
+func TestUnregisteredReadsAndSessionRecovery(t *testing.T) {
```

Comment:
Integration tests must have `Integration` in their name otherwise they will never be executed (see [Makefile](https://github.com/influxdata/telegraf/blob/c31d402b7295e07a7279da8d16bf9f93367d98ba/Makefile#L154))!

Same for the other integration test below...